.github/workflows/security-testing.yml

```
@@ -173,3 +85,42 @@ jobs:
+
+    - name: Upload test results
+      uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: container-test-results
+        path: |
+          tests/e2e/test-results/
+
+    - name: Test summary
+      if: always()
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)